src/upstream_patches_ui.sh

```diff
@@ -193,6 +200,35 @@ function show_bookmarked_series_details()
   esac
 }
 
+# Screen that shows all types of settings available.
+function show_settings_screen()
+{
+  local -a menu_list_string_array
+  local message_box
+  local ret
+
+  menu_list_string_array=('Mailing Lists')
```

Comment:
I think the name `Mailing List` is not good since it does not give the action idea. Maybe we can try something like `Follow Mailing List` or similar.